src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -296,6 +303,92 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
             return default;
         }
 
+        private static ImmutableArray<TaggedText> GetConstantValueContent(
+            Workspace workspace,
+            SyntaxToken token,
+            ISyntaxFactsService syntaxFacts,
+            SemanticModel semanticModel,
+            CancellationToken cancellationToken)
+        {
+            if (!syntaxFacts.IsBinaryExpression(token.Parent))
+            {
+                return default;
+            }
+
+            syntaxFacts.GetPartsOfBinaryExpression(token.Parent, out var leftExpression, out var operatorToken, out var rightExpression);
+
+            if (token != operatorToken)
+            {
+                return default;
+            }
+
+            var constant = semanticModel.GetConstantValue(token.Parent, cancellationToken);
+            if (!constant.HasValue)
+            {
+                return default;
+            }
+
+            var leftConstant = semanticModel.GetConstantValue(leftExpression, cancellationToken);
+            if (!leftConstant.HasValue)
+            {
+                return default;
+            }
+
+            var rightConstant = semanticModel.GetConstantValue(rightExpression, cancellationToken);
+            if (!rightConstant.HasValue)
+            {
+                return default;
+            }
+
+            var generator = SyntaxGenerator.GetGenerator(workspace, semanticModel.Language);
+
+            var textBuilder = new List<TaggedText>();
+            textBuilder.AddLineBreak();
+            textBuilder.AddText(FeaturesResources.Constant_value_colon);
+            textBuilder.AddSpace();
+            AddConstantText(leftConstant.Value);
+            textBuilder.AddSpace();
+            textBuilder.AddOperator(operatorToken.Text);
+            textBuilder.AddSpace();
+            AddConstantText(rightConstant.Value);
+            textBuilder.AddSpace();
+            textBuilder.AddOperator("=");
+            textBuilder.AddSpace();
+            AddConstantText(constant.Value);
+            return textBuilder.ToImmutableArray();
+
+            void AddConstantText(object value)
+            {
+                textBuilder.AddRange(TrimTaggedTextRun(LiteralDisplayBuilder.Build(syntaxFacts, generator, value), 42));
```

Comment:
why 42?

Reply:
It looked just right.

Reply:
If we end up not showing the binary operator with both operands and only the result, it may be appropriate to make this larger.